### ISSUE TITLE
Bactopia-tools ariba: ariba-summary error

### ISSUE THREAD

**Issue by alienzj** [bactopia/bactopia#341]
```
Including 10 samples for analysis
Found 10 samples to process

If this looks wrong, now's your chance to back out (CTRL+C 3 times).
Sleeping for 5 seconds...
--------------------------------------------------------------------
[65/3b0b1c] process > BACTOPIATOOLS:ARIBA:ARIBA_RUN (E10 - card) [100%] 10 of 10, cached: 10 ✔
[-        ] process > BACTOPIATOOLS:ARIBA:CSVTK_CONCAT_REPORT    -
[-        ] process > BACTOPIATOOLS:ARIBA:CSVTK_CONCAT_SUMMARY   -
[-        ] process > BACTOPIATOOLS:CUSTOM_DUMPSOFTWAREVERSIONS  -
Error executing process > 'BACTOPIATOOLS:ARIBA:CSVTK_CONCAT_SUMMARY (ariba-summary)'

Caused by:
  Process `BACTOPIATOOLS:ARIBA:CSVTK_CONCAT_SUMMARY` input file name collision -- There are multiple input files for each of the following file names: summary.csv


Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line



    Bactopia Tools: `ariba Execution Summary
    ---------------------------
    Bactopia Version : 2.1.0
    Nextflow Version : 22.04.0
    Command Line     : nextflow run /home/jiezhu/.conda/envs/bactopia//share/bactopia-2.1.x/main.nf -w /lustre1/g/sph_tunlab/jiezhu/projects/aces/assay/bactopia/work/ --wf ariba --bactopia results_debug --include includes.txt.h10 --ariba_dir /home/jiezhu/databases/ecogenomics/bactopia_tools/ariba --ariba_db card -resume
    Resumed          : true
[65/3b0b1c] process > BACTOPIATOOLS:ARIBA:ARIBA_RUN (E10 - card) [100%] 10 of 10, cached: 10 ✔
[-        ] process > BACTOPIATOOLS:ARIBA:CSVTK_CONCAT_REPORT    -
[-        ] process > BACTOPIATOOLS:ARIBA:CSVTK_CONCAT_SUMMARY   -
[-        ] process > BACTOPIATOOLS:CUSTOM_DUMPSOFTWAREVERSIONS  -
Error executing process > 'BACTOPIATOOLS:ARIBA:CSVTK_CONCAT_SUMMARY (ariba-summary)'

Caused by:
  Process `BACTOPIATOOLS:ARIBA:CSVTK_CONCAT_SUMMARY` input file name collision -- There are multiple input files for each of the following file names: summary.csv


Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line
```

**Comment by rpetit3** [comment]
Thank you for reporting @alienzj  I know what the issue is here and well get you an update very soon

**Comment by rpetit3** [comment]
This is now fixed on the dev build: https://github.com/bactopia/bactopia/commit/9aad38b1eb4bdcbb93af530a5a87a8147077187f

**Comment by alienzj** [comment]
Thanks Robert !

**Comment by alienzj** [comment]
[run_bactopia_ariba.log](https://github.com/bactopia/bactopia/files/8998011/run_bactopia_ariba.log)
Hi, @rpetit3 after using bactopia-dev, I still meet some errors.

**Comment by rpetit3** [comment]
Ah this looks like something different,

```
Command error:
  [ERRO] parse error on line 2, column 256: bare " in non-quoted-field
```

Let me see if I can replicate this

**Comment by anwarMZ** [comment]
@rpetit3 any update on this? 

> d error:
  [ERRO] parse error on line 2, column 256: bare " in non-quoted-field

I am getting same error ` [ERRO] parse error on line 239, column 258: bare " in non-quoted-field`

**Comment by anwarMZ** [comment]
Does merged reports include all samples?

**Comment by rpetit3** [comment]
Found the issue, and fixed in https://github.com/bactopia/bactopia/commit/9d0d676b2ba0847bc2dfdc9083c1d203ea664e40. This fix is now available on my dev build.

```
mamba create -n bactopia-dev -c rpetit3 -c conda-forge -c bioconda bactopia
conda activate bactopia-dev
``` 

Details about the issue:

Ariba outputs some strings like so:
```
b"APH(3')-IIIa is a plasmid-encoded aminoglycoside phosphotransferase in S. aureus and Enterococcus spp.";APH(3')-IIIa
```
csvtk does not like mixing double and single quotes in a columns, but you can use `--lazy-quotes` to allow this behavior.

I'm going to allow this behavior, since its what Ariba has given us to work with. Please let me know if it causes you any issues.

Robert

**Comment by rpetit3** [comment]
Hi @alienzj  and @anwarMZ 

I've released [v2.1.1](https://github.com/bactopia/bactopia/releases/tag/v2.1.1) which should fix this issue. If not please feel free to reopen.

Cheers,
Robert
